provider/source/data/segmenter/uprops/small/ExtPict.toml

```diff
@@ -5,8 +5,8 @@
 #
 # machine-generated by: icuexportdata.cpp
 
-icu_version = "74.2"
```

Comment:
@sffc Updating this data requires updating a git tag somewhere, yes?

Reply:
The property files  is picked up from icuexport data that is we use now by provider/source/tests/data/icuexport.

Reply:
I believe `provider/source/data/segmenter` is not synced with anything because it is aligned with the version of Unicode Segmentation that ICU4X currently supports, which lags behind the ICU4C version. @robertbastian can confirm.

Reply:
Ah, I see. That's fine, then!